From: Maximiliano Curia <maxy@gnuservers.com.ar>
Date: Sun, 26 Mar 2017 19:30:39 +0200
Subject: Drop html markup from polkit action file

The html markup is not allowed in the polkit action files and produces a
warning.

Closes: 696905
---
 processcore/actions.actions | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/processcore/actions.actions b/processcore/actions.actions
index ebf0d15..0f5e5e3 100644
--- a/processcore/actions.actions
+++ b/processcore/actions.actions
@@ -296,7 +296,7 @@ Description[fy]=Feroaret de leaflikheid fan in oantsjutte proses
 Description[ga]=Athraigh tosaíocht próisis áirithe
 Description[gl]=Troca o valor nice dun proceso dado
 Description[he]=שינוי את ה־niceness של תהליך נתון
-Description[hr]=Promijeni procesorski udio (<i>nice</i>) za dani proces
+Description[hr]=Promijeni procesorski udio (nice) za dani proces
 Description[hu]=Egy processzus prioritásának (nice) módosítása
 Description[ia]=Cambia le sympathia (prioritate) de un date processo
 Description[id]=Ubah kebaikan proses yang diberikan
